src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Machine {
+    public void doWork() {

```

Comment:
Incorrect grammar in the message. Change 'it is' to 'its'. Example: 'Machine started its work'